kuma/static/styles/minimalist/structure/_footer.scss

```diff
@@ -1,183 +1,144 @@
+    color: $text-color-inverted;
+    padding: 40px 20px;
 
-    a {
-        display: inline-block;
-        padding: ($grid-spacing / 2) 0;
+    @media #{$mq-tablet-and-up} {
+        padding: 40px;
     }
-}
 
-.footer-social {
-    @include set-font-size($h4-font-size);
-    display: inline-block;
-    /* prettier-ignore */
-    @include bidi(((margin-right, ($grid-spacing / 2 ), margin-left, 0),));
+    .content-container {
+        margin: 0 auto;
+        max-width: 1400px;
+    }
 
     a {
-        /* prettier-ignore */
-        @include bidi(((padding-right, $grid-spacing, padding-left, 0),));
+        color: $text-color-inverted;
     }
-}
-
-.languages {
-    @include clearfix();
-    border-bottom: 1px solid #fff;
-    padding-bottom: $mobile-center-spacing;
-    margin-bottom: $mobile-center-spacing;
 
-    label {
-        @include set-font-size($tiny-font-size);
+    ul {
+        a {
+            display: block;
+            padding: 10px 0;
+        }
     }
-}
 
-.footer-tos {
-    @include set-font-size($base-bump-font-size);
-    margin-bottom: $mobile-center-spacing;
+    .social {
+        display: flex;
+        align-items: center;
 
-    li {
-        display: inline-block;
-        /* prettier-ignore */
-        @include bidi(((margin-right, $grid-spacing, margin-left, 0),));
-    }
-}
+        h4 {
+            margin: inherit;
+            margin-right: 10px;
+            font-weight: normal;
+        }
 
-@media #{$mq-tablet-and-up} {
-    .nav-footer {
-        padding-top: $gutter-width;
-        padding-bottom: $gutter-width;
-    }
+        li {
+            display: inline-block;
+            margin-right: 10px;
+        }
 
-    .nav-footer-logo,
-    .nav-footer-mozilla {
-        margin-bottom: $gutter-width;
+        svg {
+            width: 18px;
+            height: 18px;
+            vertical-align: sub;
+        }
     }
 
-    .footer-title {
-        display: none;
+    .footer-license {
+        margin: 10px 0;
+
+        @media #{$mq-tablet-and-up} {
+            margin: 50px 0 65px;
+            padding: 50px 0 0;
+            border-top: 1px solid $border-color;
+        }
     }
 
-    .footer-group {
-        padding-bottom: $gutter-width;
-        margin-bottom: $gutter-width;
+    .footer-legal {
+        @media #{$mq-tablet-and-up} {
+            padding-bottom: 40px;
+        }
 
-        a {
-            padding: ($grid-spacing / 4) 0;
+        li {
+            display: inline-block;
         }
-    }
 
-    .footer-social {
         a {
-            /* prettier-ignore */
-            @include bidi(((padding-right, ($grid-spacing / 2 ), padding-left, 0),));
+            display: block;
+            padding: 0 20px 0 0;
+            text-decoration: underline;
         }
     }
+}
 
-    .languages {
-        @include clearfix();
-        padding-bottom: $gutter-width;
-        margin-bottom: $gutter-width;
+@supports (display: grid) {
+    .content-container {
+        display: grid;
```

Comment:
I don't have much experience with CSS grids, so this is very neat to see how it's implemented !